test/utils/P256.t.sol

```diff
@@ -14,6 +14,8 @@ contract P256Test is Test {
     using BytesLib for bytes;
     using stdJson for string;
 
+    uint256 private constant _N =
```

Comment:
maybe u can pull this from the contract instead?

Reply:
It's an `internal` `constant` in my `p256` contract and thus there is no getter for it.